src/modules/job-manager/event.c

```diff
@@ -280,16 +300,20 @@ int event_job_update (struct job *job, const char *event)
         if (job->state == FLUX_JOB_RUN)
             job->state = FLUX_JOB_CLEANUP;
     }
+    json_decref (o);
     return 0;
 inval:
     errno = EINVAL;
 error:
+    save_errno = errno;
+    json_decref (o);
+    errno = save_errno;
     return -1;
 }
 
-int event_job_post (struct event_ctx *ctx, struct job *job,
-                    flux_continuation_f cb, void *arg,
-                    const char *name, const char *context)
+static int post (struct event_ctx *ctx, struct job *job,
```

Comment:
What's the reason for this change?  It seems like we've created the `post()` function without gaining anything?

Reply:
doh, I had changed `event_job_post()` at one point, thus making the function wrapper for `post()`.